### PR TITLE
fix: component modal address

### PR DESCRIPTION
**Pull request by MNAlfiansyah**
## Changes

- fixing modal address

## Evidence
title: Component Modal Address
project: Jabar Super Apps
participants: @rayfajars @johndy2742 @ristianaditya @MNAlfiansyah

### CODE REVIEW

File: dashboard/components/Base/Modal/FormAddress.vue [hunk 1]
```diff
@@ -40,7 +40,7 @@
                   >
                     <template #label>
                       <spans
-                        v-if="state.selectedProvince.id === undefined"
+                        v-if="state.selectedProvince?.id === undefined"
```
Comment (rayfajars) [inline dashboard/components/Base/Modal/FormAddress.vue:43]: same thing with javascript truthy

File: dashboard/components/Base/Modal/FormAddress.vue [hunk 2]
```diff
@@ -53,15 +53,15 @@
                 <UFormGroup label="Kota/Kabupaten" class="mb-4 w-full">
                   <USelectMenu
                     v-model="state.selectedCity"
-                    :disabled="!!state.selectedProvince.values"
+                    :disabled="state.selectedProvince?.id === undefined"
```
Comment (rayfajars) [inline dashboard/components/Base/Modal/FormAddress.vue:56]: same thing

File: dashboard/components/Base/Modal/FormAddress.vue [hunk 3]
```diff
@@ -53,15 +53,15 @@
...
                     :options="state.dataCity"
                     searchable
                     @click="getCity"
                     @change="changeCity"
                   >
                     <template #label>
                       <span
-                        v-if="state.selectedCity.id === undefined"
+                        v-if="state.selectedCity?.id === undefined"
```
Comment (rayfajars) [inline dashboard/components/Base/Modal/FormAddress.vue:64]: same thing

**Review by rayfajars — APPROVED** [review]

**Review by rayfajars — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

File: dashboard/components/Base/Modal/FormAddress.vue (hunk 1 above)
Comment (MNAlfiansyah) [reply dashboard/components/Base/Modal/FormAddress.vue]: updated kang [5e17a66](https://github.com/jabardigitalservice/event-management-system/pull/83/commits/5e17a6694daadf601d9b294c3c77e48a28900110)

File: dashboard/components/Base/Modal/FormAddress.vue (hunk 2 above)
Comment (MNAlfiansyah) [reply dashboard/components/Base/Modal/FormAddress.vue]: updated kang [5e17a66](https://github.com/jabardigitalservice/event-management-system/pull/83/commits/5e17a6694daadf601d9b294c3c77e48a28900110)

File: dashboard/components/Base/Modal/FormAddress.vue (hunk 3 above)
Comment (MNAlfiansyah) [reply dashboard/components/Base/Modal/FormAddress.vue]: updated kang [5e17a66](https://github.com/jabardigitalservice/event-management-system/pull/83/commits/5e17a6694daadf601d9b294c3c77e48a28900110)

**Review by marsellavaleria19 — APPROVED** [review]

**Pull request merged**